services/api-server/Dockerfile

```diff
@@ -118,18 +119,27 @@ WORKDIR /home/scu
 # Starting from clean base image, copies pre-installed virtualenv from prod-only-deps
 COPY --chown=scu:scu --from=prod-only-deps  ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 
+# wget required by wait-for
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  wget \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
 # Copies booting scripts
 COPY --chown=scu:scu services/api-server/docker services/api-server/docker
 RUN chmod +x services/api-server/docker/*.sh
+COPY --chown=scu:scu scripts/common-docker-boot/wait-for /usr/local/bin/wait-for
+RUN chmod +x /usr/local/bin/wait-for
 
 HEALTHCHECK --interval=30s \
-    --timeout=20s \
-    --start-period=30s \
-    --retries=3 \
-    CMD ["python3", "services/api-server/docker/healthcheck.py", "http://localhost:8000/"]
+  --timeout=20s \
+  --start-period=30s \
+  --retries=3 \
+  CMD ["python3", "services/api-server/docker/healthcheck.py", "http://localhost:8000/"]
 
 ENTRYPOINT [ "/bin/sh", "services/api-server/docker/entrypoint.sh" ]
-CMD ["/bin/sh", "services/api-server/docker/boot.sh"]
+CMD ["/bin/sh", "-c",  "wait-for http://migration:8000 -- services/api-server/docker/boot.sh"]
```

Comment:
We are going to have an issue in the deployments here.
all the apps are renamed using ```$SWARM_STACK_NAME_migration```. Please double-check and correct as this will fail for sure.